README.md

```diff
@@ -1,11 +1,50 @@
-# README
+# Postcoder ![Travis CI build status](https://travis-ci.org/fivegoodfriends/postcoder.svg?branch=master)
```

Comment:
Woohoo, build badge!

Reply:
Can you make this link to the actual build? Like https://travis-ci.org/fivegoodfriends/postcoder

Reply:
Done, force pushed in preparation for merge.